.changeset/fair-vans-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+'react-select': patch
+---
+
+Move emotion packages and types for react-transition-group to peerDependencies
```

Comment:
Is it still `patch` if a dependency has been moved to peerDependencies?

Reply:
Code that ships to the user has been changed, but it doesn't add any new functionality (minor) or change the existing API (major), so I'd consider it a patch (unless I'm missing something?)

Reply:
They might not already have `emotion` explicitly added as a dependency in their application, so wouldn't it be a potentially breaking change?

Reply:
Thats a good point. 
I think it varies by package manager, so it _might_ technically be a breaking change. I'll try to find someone who can give us a definitive answer.